.all-contributorsrc

```diff
@@ -232,7 +232,8 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/70570907?v=4",
       "profile": "https://github.com/chouchouji",
       "contributions": [
-        "code"
+        "code",
+        "maintenance"
```

Comment:
Going off of https://allcontributors.org/docs/en/emoji-key, I don't think `maintenance` is technically the right contribution type here? I see `maintenance` as reviewing PRs, triaging issues, and the like.

If not `code`, maybe - `tool`?

Reply:
Yeah, that makes sense. I'm still getting to know the categories. Good call